league_rpc_linux/__main__.py

```diff
@@ -1,6 +1,8 @@
 import argparse
 import sys
 import time
+from multiprocessing import Process
```

Comment:
What's the reason for picking multiprocessing here instead of threading

Reply:
I'm not really too familiar with either one, for me it seemed more logical to use multiprocessing, as its logically running separated from the main with the websocket. 
I guess threading wouldve been also a viable option, thats beyond my knowledge though. 
So not really a thoughtful decision, but a working one..